src/dataframe/dataframe.jl

```diff
@@ -768,21 +782,21 @@ function insertcols!(df::DataFrame, col_ind::Int; makeunique::Bool=false, name_c
 end
 
 
-##############################################################################
-##
-## Copying
-##
-##############################################################################
-
-# A copy of a DataFrame points to the original column vectors but
-#   gets its own Index.
-Base.copy(df::DataFrame) = DataFrame(copy(_columns(df)), copy(index(df)))
+"""
+    copy(df::DataFrame; copycolumns::Bool=true)
 
-##############################################################################
-##
-## Deletion / Subsetting
-##
-##############################################################################
+    Copy a `DataFrame` `df`.
```

Comment:
Wrong indentation. I suggest something like this:
```
Copy data frame `df`.
If `copycolumns=true` (the default), return a new  `DataFrame` holding
copies of column vectors in `df`.
If `copycolumns=false`, return a new `DataFrame` sharing column vectors with `df`.

Reply:
OK